.github/workflows/linter-conan-v2.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   test_linter:
-    name: Test linter changes
+    name: Test linter changes (v2 migration)
```

Comment:
We han two actions with the same `Test linter changes` name. In the configuration file we want to identify them by name (better than id)